client/flutter/lib/module_home/service/home_service.dart

```diff
@@ -1,9 +1,57 @@
 import 'package:injectable/injectable.dart';
+import 'package:twaslna_delivery/generated/l10n.dart';
 import 'package:twaslna_delivery/module_home/manager/home_manager.dart';
+import 'package:twaslna_delivery/module_home/response/products.dart';
+import 'package:twaslna_delivery/module_home/response/store_categories.dart';
+import 'package:twaslna_delivery/utils/models/product.dart';
+import 'package:twaslna_delivery/utils/models/store_category.dart';
 
 @injectable
 class HomeService {
   HomeManager _homeManager;
 
   HomeService(this._homeManager);
+
+  Future getTopProducts() async {
+    Products? topProducts = await _homeManager.getTopProducts();
+    if (topProducts == null) return null;
+    if (topProducts.msgErr != null) {
+      return topProducts.msgErr;
+    }
+    List topProductsModel = <ProductModel>[];
+    topProducts.data?.forEach((element) {
+      topProductsModel.add(ProductModel(
+          title: element.productName ?? S.current.product,
+          image:
+              'https://www.erdeundleben.com/wp-content/uploads/2021/02/folgendes-macht-unser-food-personal-wenn-es-fast-zu-mude-ist-um-zu-kochen-0-Yywyr8ju.jpg',
```

Comment:
same assets note

Reply:
for models I created very high protected way to handle data and recover when there is any kind of missed values 
so for images I will create image provider has already static images  spirited  than asset images because assets will make 
apk size bigger